server/api/crud/model_monitoring/model_endpoints.py

```diff
@@ -13,7 +13,6 @@
 # limitations under the License.
```

Comment:
In this module:
Please update the docstrings of `ModelEndpoints.create_model_endpoint` - it still mentions only `mlrun.config.model_endpoint_monitoring.store_type` as the store type selector.